src/main/java/Ui.java

```diff
@@ -0,0 +1,78 @@
+package main.java;
+
+import java.util.Scanner;
+
+import tasks.Task;
+
+/**
+ * Ui class handles the user interface of the application.
+ * It is responsible for reading user input and printing messages to the user.
+ */
+public class Ui {
+
+    private Scanner sc;
+
+    /*
+     * Constructor for Ui class.
+     * It initializes the scanner object to read user input.
+     * It also prints a greeting message to the user.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+        this.greet();
+    }
+
+    /*
+     * Prints a greeting message to the user when the application starts.
+     */
+    public void greet() {
+        String logo = "▀█▀ ▄▀█ █▀ █▄▀ █▄█ ▄▀█ █▀█ █▀█ █▀▀ █▀█\n"
+                + "░█░ █▀█ ▄█ █░█ ░█░ █▀█ █▀▀ █▀▀ ██▄ █▀▄\n";
```

Comment:
I like the indentation of eight space instead of four, making the long line clean and readable.
By the way, this looks so cool! 🤩